features/pre-post-transformation/README.md

```diff
@@ -0,0 +1,119 @@
+---
+description: Request transformations on media before or after upload.
+---
+
+# Pre and Post Transformation
+
+Using ImageKit.io, you can request a transformation on image or video files when they're uploaded through the [Upload API](../../api-reference/upload-file-api/README.md).
+
+To know more about transformations, see:
+
+- [Image Transformations](../image-transformations/README.md)
+- [Video Transformations](../video-transformation/README.md)
+
+## Pre Transformation
+
+A pre-transformation is applied before the uploaded file is saved.
```

Comment:
before the file is uploaded in the media library.

Reply:
Done in https://github.com/imagekitio/docs-keep-a-changelog/pull/201/commits/376303b5a5e6c3e690d5cf118c9b131ea186fb65